tests/realtikvtest/sessiontest/BUILD.bazel

```diff
@@ -2,7 +2,7 @@ load("@io_bazel_rules_go//go:def.bzl", "go_test")
 
 go_test(
     name = "sessiontest_test",
-    timeout = "moderate",
+    timeout = "long",
```

Comment:
long indicates that this test can run for a maximum of 15 minutes. so after this PR, this test needs more time to complete. It sounds a bit odd.